# txt2c runs (only) on the build system.  This is a fairly horrible
# hack to make it compile with the build system compiler, not the
# target compiler.

--- monotone-0.19/Makefile.am.orig	2005-07-04 23:39:44.420722558 -0700
+++ monotone-0.19/Makefile.am	2005-07-04 23:36:50.267121331 -0700
@@ -356,7 +356,7 @@
 CLEANFILES = $(BUILT_SOURCES_CLEAN) $(EPS_FIGURES)
 
 txt2c: txt2c.cc Makefile
-	$(CXX) -Wall -o $@ $<
+	$(BUILD_CXX) -Wall -o $@ $<
 	chmod 0755 $@$(EXEEXT)
 
 apidocs:
